src/ota/ota_pal_test.c

```diff
@@ -399,14 +414,21 @@ TEST( Full_OTA_PAL, otaPal_WriteBlock_WriteSingleByte )
 
 /**
  * @brief Write many blocks of data to a file opened in the device. Verify success.
+ * Because of Flash property, we might not able to write same page multiple times.
```

Comment:
Add a blank comment line between brief and description.
```suggestion
 *
 * Because of Flash property, we might not able to write same page multiple times.
```